.github/workflows/ci.yaml

```diff
@@ -18,6 +18,11 @@ jobs:
       - run: node --version
       - run: npm install
       - run: npm test
+      - name: coverage
+        uses: codecov/codecov-action@v1
+        with:
+          name: actions ${{ matrix.node }}
+          fail_ci_if_error: false
```

Comment:
How is the failure surfaced if CI doesnt fail on erroor?

Reply:
Removed this. 

Google Client Libraries team uses "fail_ci_if_error: false" because of codecov flakiness, but they send out a lot more PRs than we do, so probably less of an issue for us.